content/en/docs/features/control-plane/entity-tag.md

```diff
@@ -109,15 +109,19 @@ To identify the specific resource ETAG used in the batch ETAG response, the clie
 
 #### Incremental Changes
 
-When making a GET request, clients can include ETAGs with the query parameter `incChanges` to request incremental changes from the device. Each ETAG provided by the client corresponds to a different resource ETAG.
+To request incremental changes from a device through a GET request, clients can include an `ETAG-0` in the option, along with multiple query parameters using the following format: `incChanges=<ETAG-1 in base64>..<ETAG-20 in base64>, incChanges=<ETAG-21 in base64>..<ETAG-40 in base64>, incChanges=...`. Each `ETAG` provided by the client corresponds to a different resource's `ETAG`, and the `ETAG-0` in the option represents the latest `ETAG` among all resources.
```

Comment:
include ETAG in the option? Please be more specific. CoAP option, link to RFC. Readers don't know CoAP as much as you do. Example for our client would be also beneficial. How to do it through our client API, separate subsection.

Reply:
fixed